packages/e2e-tests/specs/block-deletion.test.js

```diff
@@ -20,10 +22,17 @@ const addThreeParagraphsToNewPost = async () => {
 	await page.keyboard.press( 'Enter' );
 };
 
-const clickOnBlockSettingsMenuItem = async ( buttonLabel ) => {
-	await expect( page ).toClick( '.block-editor-block-settings-menu__toggle' );
-	const itemButton = ( await page.$x( `//*[contains(@class, "block-editor-block-settings-menu__popover")]//button[contains(text(), '${ buttonLabel }')]` ) )[ 0 ];
-	await itemButton.click();
+/**
+ * Note Puppeteer has a problem clicking on items in the Popover that require
+ * scrolling the Popover to be visible. To work around this we use the keyboard
+ * to select the "Remove Block" button
+ */
+const clickOnBlockSettingsMenuRemoveBlockButton = async () => {
+	await clickBlockToolbarButton( 'More options' );
+
+	await pressKeyTimes( 'Tab', 7 );
```

Comment:
This feels very fragile and could break pretty easily. I get that the PR shouldn't be blocked by this unrelated failure though. Maybe we could base this on the "text" of the buttons instead. (tab until you find the "remove button")

Reply:
Agreed. I felt that too.

I'll write a loop to test for the correct button before it gets pressed. This should at least make the test easier to fix if and when it does break.

Reply:
I've just pushed a fix for this.